src/test/unit/rx_spi_expresslrs_telemetry_unittest.cc

```diff
@@ -223,7 +223,7 @@ TEST(RxSpiExpressLrsTelemetryUnitTest, TestFlightMode)
 TEST(RxSpiExpressLrsTelemetryUnitTest, TestMspVersionRequest)
 { 
     uint8_t request[15] = {238,12,122,200,234,48,0,1,1,0,0,0,0,128, 0};
-    uint8_t response[12] = {200,10,123,234,200,48,3,1,0,1,44,42};
+    uint8_t response[12] = {200,10,123,234,200,48,3,1,0,1,45,255};
```

Comment:
Unfortunately I have little to no confidence in this test change I made. (My bad for not catching the failing test before pushing).

My understanding (which might be wrong) is that the test is dependent on the current MSP version (hence `response[10]`) and that `response[11]` is a checksum which also changes. I will try to go through the related ExpressLRS code to confirm that.

I assume that the MSP `API_VERSION_MINOR` can be used instead of `response[10]` and then  `response[11]` can be computed on the fly to prevent this test from failing with future MSP version updates.